tests/unit/sinonMongoose.js

```diff
@@ -20,6 +18,7 @@ const stubChainedQueries = (stubbedMethodReturns, chainedQueries = ['populate',
 
 const checkFirstQueryCoherence = (stubbedMethod, chainedPayload, callCount) => {
   const expectedQuery = chainedPayload[0].query;
+  if (!stubbedMethod.getCall(callCount)) sinon.assert.fail(`"${stubbedMethod}" stub not called on call ${callCount}`);
```

Comment:
Que penses-tu comme message d'erreur de `"${stubbedMethod}" is not called ${callCount + 1} times` ?

Reply:
done